compiler/src/main/scala/com/fortysevendeg/exercises/compiler/TreeGen.scala

```diff
@@ -87,8 +87,13 @@ case class TreeGen[U <: Universe](
   }
 
   def makeLibrary(
-    name: String, description: String, color: Option[String],
-    sectionTerms: List[TermName], owner: String, repository: String
+    name:         String,
+    description:  String,
+    color:        Option[String],
+    sectionTerms: List[TermName],
+    owner:        String,
+    repository:   String,
+    timestamp:    String
```

Comment:
We just need a string timestamp so i think it's overkill. We've used strings for contribution timestamps too.